source/administration/migrating.rst

```diff
@@ -89,13 +89,13 @@ If your data in the bespoke messenger is vital, consider:
 
 1. `Mattermost Bulk Load tool <https://docs.mattermost.com/deployment/bulk-loading.html>`__ - Use the Mattermost Bulk Load tool to ETL from your bespoke system to Mattermost.
 2. `Mattermost ETL framework from BrightScout <https://github.com/Brightscout/mattermost-etl>`__- Consider the Mattermost ETL framework from BrightScout to custom-configure an adapter to plug in to the Bulk Load tool mentioned above.
```

Comment:
Why are the first to bullet points not formatted the same way as the below ones?

Reply:
Good question - I didn't actually notice that! I can't seem to add bolding without messing up the link formatting, but I have changed the dash to colons.